ci/gpu/build.sh

```diff
@@ -239,9 +239,10 @@ fi
 # TEST - Run py.test, notebooks
 ################################################################################
 
-cd "$WORKSPACE/python/cudf"
+cd "$WORKSPACE/python/cudf/cudf"
+# It is essential to cd into $WORKSPACE/python/cudf/cudf as `pytest-xdist` + `codecov` seem to work only at this directory level.
```

Comment:
`coverage` is the plugin that measures coverage. `codecov` is just the service we use to upload and visualize the results.
```suggestion
# It is essential to cd into $WORKSPACE/python/cudf/cudf as `pytest-xdist` + `coverage` seem to work only at this directory level.
```